project/accounts/views.py

```diff
@@ -97,104 +124,56 @@ def get_initial(self):
                 "first_name": profile.first_name or None,
                 "last_name": profile.last_name or None,
                 "about_me": profile.about_me or None,
+                "profile_image": profile.profile_image or None,
             }
         )
         return super(SettingsView, self).get_initial()
 
 
-class ProfileActivationView(View):
-    """
-    This shows different views to the user when they are verifying
-    their account based on whether they are already verified or not.
-    """
```

Comment:
Why do we remove profile activation? Then someone can register by using a fake email address and can create threads without any number limitation.  Or, voting can be manipulated by these fake accounts.

I think removing ProfileActivationView will cause severe security problems in the future.

Reply:
I'll restore that view. I mistakingly thought it was related to the beta blocker or setup wizard; I can't remember precisely the source of my confusion.

Reply:
I restored ProfileActivationView with some minor refactoring.